core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -1958,6 +1958,7 @@ SDDkwd__(EXE_DIAGNOSTIC_EVENTS,		"OFF"),
 
   DDkwd__(HIVE_DEFAULT_CHARSET,            (char *)SQLCHARSETSTRING_UTF8),
   DD_____(HIVE_DEFAULT_SCHEMA,                  "HIVE"),
+  DD_____(HIVE_FILE_CHARSET,            (char *)SQLCHARSETSTRING_UTF8),
```

Comment:
The default value for HIVE_FILE_CHARSET should not be UTF8, since that would be wrong if we have a file with ISO8859-1 characters and set HIVE_DEFAULT_CHARSET to ISO88591. Maybe better to set it to an empty string, which would mean that HIVE_FILE_CHARSET is the same as HIVE_DEFAULT_CHARSET. Given the way we implement it, maybe we should only allow an empty string and GBK as values for HIVE_FILE_CHARSET.